core/src/test/java/nl/nn/adapterframework/senders/MailSender2Test.java

```diff
@@ -12,7 +12,7 @@
 import jakarta.mail.internet.MimeMessage;
 import nl.nn.adapterframework.stream.Message;
 
-public class MailSenderTest2 extends SenderTestBase<MailSender> {
+class MailSender2Test extends SenderTestBase<MailSender> {
```

Comment:
Ik denk dat de oude naam beter is omdat het niet een test is voor MailSender2, maar de 2de test voor MailSender.
(Ik weet niet waarom daar 2 test classes voor zijn en of die samen te voegen zijn).

Reply:
Volgens mij is het wel een vereisde voor Junit dat een test klasse begint of eindigd met Test.

Reply:
Klopt en dat is precies de reden waarom ik deze aanpassing heb gedaan. Hij eindigde op 2 ipv Test. ;-)

Reply:
Klopt in theorie maar de test werd wel altijd uitgevoerd. Dus daarom dat ik dacht de oude naam misschien toch te handhaven.

Maar zo werkt het zeker dus kan ook zo blijven.

Reply:
Ah bijzonder... Maar misschien ligt dat aan het feit dat die met "Test*" begint waardoor het ook matcht.

Reply:
Ja, heb het maar effe opgezocht: Hier staat volgens mij het antwoord:
https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html
Waarbij wij ook nog alle Java files meenemen..dus dan maakt het niet uit. Echter wijken we wel af vd standaard en sourced hij ook hulp classes. 
```
<includes>
	<include>**/*.java</include>
</includes>
```

Reply:
Ah, dus iemand had zich niet aan de standaard willen houden, en toen maar naar omwegen gezocht... Klinkt bekend!

Reply:
Mmm, hopelijk kunnen we de classes met tests er in weer netjes aan de naming convention laten houden. Dan kunnen we die include weer verwijderen.  :-)